dev/packages/alpha/cisco-0.0.2/dataset/asa/elasticsearch/ingest-pipeline/default.yml

```diff
@@ -0,0 +1,1281 @@
+---
+description: "Pipeline for Cisco {< .internal_PREFIX >} logs"
```

Comment:
@ruflin not sure what to do with such entries

Reply:
Me neither. How does this work in Beats @adriansr , I see it was introduced in https://github.com/elastic/beats/pull/13286 ? Is this to reuse some ingest pipeline?

For now we should just type it out.

Reply:
So this pipeline is shared between cisco/asa and cisco/ftd, because the ftd has to support all the same messages as the asa plus an extra few ones. The `.internal_prefix` being `asa` or `ftd` used to define the final location for fields in a document (`cisco.asa.xxx` or `cisco.ftd.xxx`). `.internal_PREFIX` it's just the upper-case equivalent, `ASA` and `FTD`.

In retrospect this could've been implemented by separating the common part into a different pipeline instead of one big templated pipeline.

Reply:
I doubt we will support reusing pipelines in packages in the near term. Instead I think this kind of logic / magic should happen in the future where the packages are developed (I know, today it is the same place). This will keep the package format and logic in EPM simple.

So for now, lets just hard code it / copy it.